From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Developer <developer@eclipse.org>
Date: Mon, 6 Jan 2026 12:00:00 +0000
Subject: [PATCH] Export targets for external use

The try_atomic_add in score/language/safecpp/safe_atomics is currently
restricted. This patch makes it available to external consumers.

This is required by score_communication that depends on score_baselibs.
---
 score/language/safecpp/safe_atomics/BUILD | 1 +
 1 file changed, 1 insertion(+)

diff --git a/score/language/safecpp/safe_atomics/BUILD b/score/language/safecpp/safe_atomics/BUILD
index 1234567..abcdefg 100644
--- a/score/language/safecpp/safe_atomics/BUILD
+++ b/score/language/safecpp/safe_atomics/BUILD
@@ -20,6 +20,7 @@ cc_library(
     tags = ["FFI"],
     visibility = [
         "@score_baselibs//score/language/safecpp/safe_math:__pkg__",
+        "//visibility:public",
     ],
     deps = [
         ":error",
-- 
2.40.0
